src/utilities/gql.ts

```diff
@@ -204,6 +204,7 @@ export enum Queries {
   USERS = 'users',
   VALIDATE_ACTIVITY_ARGUMENTS = 'validateActivityArguments',
   VIEW = 'view_by_pk',
+  VIEW_TO_MISSION_MODEL = 'view_to_mission_model',
```

Comment:
is this being used?

Reply:
Old code from when it was it's own table, I'll delete!